br/pkg/lightning/importer/dup_detect.go

```diff
@@ -105,38 +108,59 @@ func makeDupHandlerConstructor(
 	}
 }
 
+// ErrDuplicateKey is an error class for duplicate key error.
+var ErrDuplicateKey = errors.Normalize("duplicate key detected on indexID %d of KeyID: %v", errors.RFCCodeText("Lightning:PreDedup:ErrDuplicateKey"))
```

Comment:
It's better to change `indexID` and `keyID` to user-friendly meaning, like `indexID` to `indexName` and `keyID` to file path and row number.

Reply:
This error is not displayed to user, just to pass a strcutured `error` to caller